.travis.yml

```diff
@@ -1,2 +1,8 @@
-language: c
-script: bin/shpec
+language: sh
+script:
+  - time bash bin/shpec -f 1 shpec/shpec_shpec.sh
+  - time dash bin/shpec -f 1 shpec/shpec_shpec.sh
+  - time ksh  bin/shpec -f 1 shpec/shpec_shpec.sh
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y ksh
```

Comment:
@hlangeveld - is it possible to split the tests up per shell in travis like this?
(see https://github.com/hlangeveld/shpec/pull/1 https://travis-ci.org/hlangeveld/shpec/builds/59175163 )